index.js

```diff
@@ -3,18 +3,11 @@
 const parse = require('./lib/xml')
 
 module.exports = (options) => {
-  return function * (next) {
-    const ctx = this
-    if (ctx.is('xml')) {
-      switch (ctx.method) {
-        case 'POST':
-        case 'PUT':
-        case 'PATCH':
-        case 'TRACE':
-        ctx.request.body = yield parse(ctx.req, options)
-        break
-      }
+    return function*(next) {
```

Comment:
I use standard coding style in a lot of projects of mine,

So I gently ask you to give space between *

```javascript
return function * (next) { }
```